Old version references in hapi-fhir-tutorial/simple-server POM causes failure

**Describe the bug**
When running "mvn clean package" at directory "hapi-fhir/hapi-fhir-tutorial/simple-server", theere is a failure

**To Reproduce**
Steps to reproduce the behavior:
1. Clone repo
2. `cd hapi-fhir-tutorial/simple-server`
3. `mvn clean package`
4. See error

```
[INFO] ------------------------------------------------------------------------
[INFO] Building HAPI FHIR Example - Simple Server 0.7
[INFO] ------------------------------------------------------------------------
[WARNING] The POM for ca.uhn.hapi.fhir:hapi-fhir-base:jar:1.2-SNAPSHOT is missing, no dependency information available
[WARNING] The POM for ca.uhn.hapi.fhir:hapi-fhir-structures-dstu:jar:1.2-SNAPSHOT is missing, no dependency information available
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 0.296 s
[INFO] Finished at: 2018-11-29T13:15:53-07:00
[INFO] Final Memory: 9M/309M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal on project hapi-fhir-example-simple-server: Could not resolve dependencies for project ca.uhn.hapi.example:hapi-fhir-example-simple-server:war:0.7: The following artifacts could not be resolved: ca.uhn.hapi.fhir:hapi-fhir-base:jar:1.2-SNAPSHOT, ca.uhn.hapi.fhir:hapi-fhir-structures-dstu:jar:1.2-SNAPSHOT: Failure to find ca.uhn.hapi.fhir:hapi-fhir-base:jar:1.2-SNAPSHOT in https://oss.sonatype.org/content/repositories/snapshots/ was cached in the local repository, resolution will not be reattempted until the update interval of oss-snapshots has elapsed or updates are forced -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/DependencyResolutionException
```

**Expected behavior**
I expect the build to be successful

**Screenshots**
If applicable, add screenshots to help explain your problem.

**Environment (please complete the following information):**
 - HAPI FHIR Version: `3.7.0-SNAPSHOT`
 - OS: Mac OSX
 - Browser: N/A

**Additional context**
I have a candidate fix that works and plan to submit a Pull Request

--- a/hapi-fhir-tutorial/simple-server/pom.xml
+++ b/hapi-fhir-tutorial/simple-server/pom.xml
@@ -14,7 +14,7 @@
 	
 	<groupId>ca.uhn.hapi.example</groupId>
 	<artifactId>hapi-fhir-example-simple-server</artifactId>
-	<version>0.7</version>
+	<version>0.8</version>
 	<packaging>war</packaging>
 
 	<name>HAPI FHIR Example - Simple Server</name>
@@ -35,12 +35,12 @@
 		<dependency>
 			<groupId>ca.uhn.hapi.fhir</groupId>
 			<artifactId>hapi-fhir-base</artifactId>
-			<version>1.2-SNAPSHOT</version>
+			<version>1.6</version>
 		</dependency>
 		<dependency>
 			<groupId>ca.uhn.hapi.fhir</groupId>
 			<artifactId>hapi-fhir-structures-dstu</artifactId>
-			<version>1.2-SNAPSHOT</version>
+			<version>1.6</version>
 		</dependency>
 
 		<!-- 

